opensearch-project/OpenSearch-Dashboards
# Update sample data page UI when useUpdatedUX enabled

### Description

This PR is for updating the sample data page UI when useUpdatedUX enabled. It mainly includes below changes:
- Move data source selector to the top right to align with other pages
- Remove duplicate sample data title

### Issues Resolved



#8312

## Screenshot


![image](https://github.com/user-attachments/assets/74e58cf3-ebb8-4286-bfc7-57ee6536bc1d)


## Testing the changes



- Clone branch code and run `yarn osd bootstrap --single-version ignore`
- Add below configs in `config/opensearch_dashboards.yml`
```
savedObjects.permission.enabled: true
workspace.enabled: true
uiSettings:
  overrides:
    'home:useNewHomePage': true
opensearchDashboards.dashboardAdmin.users: ['admin']
```
- Run `yarn start --no-base-path`
- Login with admin user and enter a specific workspace
- Click "Sample data" in the left navigation
- The UI should be the same with above screenshot

## Changelog

- feat: Update sample data page UI when useUpdatedUX enabled

### Check List
